packages/xstate-react/src/useInterpret.ts

```diff
@@ -119,6 +125,12 @@ export function useInterpret<
     };
   }, []);
 
+  useIsomorphicLayoutEffect(() => {
+    if (input) {
+      service.input(input);
```

Comment:
Without bumping the peer dep requirement we can't be sure if the installed version of XState supports this method.
```suggestion
      service.input?.(input);
```